test-suite/fhevm/docker-compose/relayer-docker-compose.yml

```diff
@@ -4,10 +4,20 @@ services:
     image: ghcr.io/zama-ai/console/fhevm-relayer:${RELAYER_VERSION}
     env_file:
       - ../env/staging/.env.relayer.local
+    environment:
+      # cheating for minio dns resolution
```

Comment:
Should we open an issue to revisit this and find a way to fetch this dynamically? @rudy-6-4